user/inc/application.h

```diff
@@ -52,7 +52,13 @@
 #include "spark_wiring_network.h"
 #include "spark_wiring_client.h"
 #include "spark_wiring_startup.h"
+
```

Comment:
I looked at putting this into spark_wiring_timer.h instead. I decided not to only because that would require a large #ifdef/#else block in there. Their HAL functions are radically different to the Nordic system, so it would only make sense to replace just about everything.

Happy to do it that way if you prefer.